src/controller.rs

```diff
@@ -237,6 +227,11 @@ impl Controller for FoxBox {
         self.certificate_manager.clone()
     }
 
+    /// Every box should have a default certificate that can be used to uniquely identify it.
```

Comment:
Let's rephrase and extend this comment:

`Every box should create a self signed certificate for the local domain.  The fingerprint of that certificate becomes the box's identifier and is used to create the public DNS zone and the local (i.e. local.<fingerprint>.knilxof.org) and remote (i.e. remote.<fingerprint>.knilxof.org) urls`